autoware_launch/launch/autoware.launch.xml

```diff
@@ -99,6 +101,7 @@
       <arg name="vehicle_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.param.yaml"/>
       <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
       <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
+      <arg name="enable_fine_detection" value="$(var enable_traffic_light_fine_detection)"/>
```

Comment:
In [universe launcher](https://github.com/autowarefoundation/autoware.universe/pull/1991), the traffic light fine detection option is called `enable_fine_detection`.